cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -311,10 +314,16 @@ Listing constraints can be done with `SHOW CONSTRAINTS`, which will produce a ta
 | `+`
 |===
 
+Listing constraints also allows for `WHERE` and `YIELD` clauses to filter the returned rows and columns.
+
 [NOTE]
 The deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the <<administration-security-administration-database-constraints, `SHOW CONSTRAINTS` privilege>>.
 
-include::example-of-listing-constraints.asciidoc[leveloffset=+2]
+==== Listing constraints examples
+
+include::listing-all-constraints.asciidoc[leveloffset=+2]
+
+include::listing-constraints-with-filtering.asciidoc[leveloffset=+2]
```

Comment:
The nesting of this sub-section is a little out, as *Listing all constraints* and *Listing constraints with filtering* should be 3.6.1.1 and 3.6.1.2.

I'm not sure how to fix this without rebuilding the docs - I can push an update to this PR though when I work it out! 😄 
![Screenshot 2021-04-08 at 14 50 31](https://user-images.githubusercontent.com/12032446/114038471-ce079400-9879-11eb-8636-e3634cbc029e.png)

Reply:
Look at how the index chapter is doing it, it might give some ideas (or have the same issue)

Reply:
Thanks @Hunterness - yes, the Indexes page has the same issue: https://neo4j.com/docs/cypher-manual/4.3-preview/administration/indexes-for-search-performance/#_listing_indexes_examples

Still having a play to find the right combination to make these sub-headers